tests/Unit/Domain/Test_DomainHelpers.cpp

```diff
@@ -437,3 +437,142 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainHelpers.AllFrustumDirections",
     }
   }
 }
+
+SPECTRE_TEST_CASE("Unit.Domain.DomainHelpers.ShellGraph", "[Domain][Unit]") {
+  std::vector<std::array<size_t, 8>> expected_corners = {
+      // Shell on left-hand side:
+      {{5, 6, 7, 8, 13, 14, 15, 16}} /*+z*/,
+      {{3, 4, 1, 2, 11, 12, 9, 10}} /*-z*/,
+      {{7, 8, 3, 4, 15, 16, 11, 12}} /*+y*/,
+      {{1, 2, 5, 6, 9, 10, 13, 14}} /*-y*/,
+      {{2, 4, 6, 8, 10, 12, 14, 16}} /*+x*/,
+      {{3, 1, 7, 5, 11, 9, 15, 13}} /*-x*/};
+  const auto generated_corners = corners_for_radially_layered_domains(1, false);
+  for (size_t i = 0; i < expected_corners.size(); i++) {
+    INFO(i);
+    CHECK(generated_corners[i] == expected_corners[i]);
+  }
+  CHECK(generated_corners == expected_corners);
```

Comment:
These checks (and several similar pairs elsewhere) seem redundant.

Reply:
I left the redundant checks in because it's convenient to have the whole list of expected corners and generated corners printed out in addition to the individual ones that failed

Reply:
OK.  That's reasonable.